src/views/ContributorRewards/components/MonthsDropdown.tsx

```diff
@@ -1,22 +1,30 @@
-import React, { useMemo } from 'react'
+import React, { useEffect, useMemo, useState } from 'react'
 import { useTheme } from 'react-neu'
 import { NavLink, useLocation } from 'react-router-dom'
 import Select from 'react-select'
 import styled from 'styled-components'
 
-const CustomOption = (props: any) => {
-  const { innerProps, value, label, data } = props
-  return (
-    <DropdownOption {...innerProps}>
-      <StyledMonth>{label}</StyledMonth>
-    </DropdownOption>
-  )
-}
-
-const MonthsDropdown: React.FC = () => {
+const MonthsDropdown = (props: { liftMonthState: Function }) => {
   const theme = useTheme()
   const { pathname } = useLocation()
 
+  const [currentMonth, setCurrentMonth] = useState('December')
+  const setMonth = (label: string) => {
+    setCurrentMonth(label)
+  }
+  useEffect(() => {
+    props.liftMonthState(currentMonth)
+  }, [props, currentMonth])
```

Comment:
same here, no need to duplicate setting state here if you can get it from the provider or via props. unless i'm missing something